lib/pages/conversation/conversation_page.dart

```diff
@@ -0,0 +1,357 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:flutter_spinkit/flutter_spinkit.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:structure_flutter/bloc/blocs/auth_provider.dart';
+import 'package:structure_flutter/core/common/constants/size_constant.dart';
+import 'package:structure_flutter/core/extension/media_extension.dart';
+import 'package:structure_flutter/core/resource/app_colors.dart';
+import 'package:structure_flutter/core/resource/icon_style.dart';
+import 'package:structure_flutter/core/resource/text_style.dart';
+import 'package:structure_flutter/data/entities/conversation.dart';
+import 'package:structure_flutter/data/entities/message.dart';
+import 'package:structure_flutter/repositories/account_repository.dart';
+import 'package:structure_flutter/repositories/storage_repository.dart';
+import 'package:timeago/timeago.dart' as timeago;
+import 'package:provider/provider.dart';
+
+class ConversationPage extends StatefulWidget {
+  String _conversationID;
+  String _receiverID;
+  String _receiverImage;
+  String _receiverName;
+
+  ConversationPage(this._conversationID, this._receiverID, this._receiverName,
+      this._receiverImage);
```

Comment:
```suggestion
      this._receiverImage,);
```
add comma and format code